pysat/tests/classes/cls_instrument_access.py

```diff
@@ -179,7 +222,17 @@ def test_basic_instrument_load_no_input(self):
                              [('fname', 'have multi_file_day and load by file'),
                               (None, 'is not supported with multi_file_day')])
     def test_instrument_load_errors_with_multifile(self, load_in, verr):
-        """Ensure load calls raises ValueError with `multi_file_day` as True."""
+        """Ensure load calls raises ValueError with `multi_file_day` as True.
+
+        Parameters
+        ----------
+        load_in : str or NoneType
+            If 'fname', load by filename. If None, load without kwargs.
```

Comment:
From memory this should only error if loading by filename. Searching for the string `test_instrument_load_errors_with_multifile` doesn't yield other uses of the method.

Reply:
I forget when this was rewritten as parametrize, but the test should have only been changed locations in this branch

Reply:
My bad, the None parameterization results in a `self.load()` call which does produce an error with multi_file_day.